### PR TITLE
Update bundle_fr.properties

### PR DESCRIPTION
**Pull request by Math2128**
Have done some more. And more to be translated later... :)
And I have no idea why everytime I do this it adds a new Patch, but yeah...

### CODE REVIEW

File: core/assets/bundles/bundle_fr.properties [hunk 1]
```diff
@@ -430,7 +430,7 @@ setting.antialias.name = Antialias[LIGHT_GRAY] (demande le redémarrage de l'app
 setting.indicators.name = Indicateurs pour les alliés
 setting.autotarget.name = Visée automatique
 setting.fpscap.name = Max FPS
-setting.fpscap.none = None
+setting.fpscap.none = aucun
```
Comment (itskatt) [inline core/assets/bundles/bundle_fr.properties]:
```suggestion
setting.fpscap.none = Aucun
```
Consistency

File: core/assets/bundles/bundle_fr.properties [hunk 2]
```diff
@@ -540,26 +540,26 @@ item.phase-fabric.name = Tissu phasé
 item.phase-fabric.description = Une substance au poids quasiment inexistant utilisé pour l'électronique avancé et la technologie auto-réparatrice.
 item.surge-alloy.name = Alliage superchargé
 item.surge-alloy.description = Un alliage avancé avec des propriétés électriques avancées.
-item.spore-pod.name = Spore Pod
-item.spore-pod.description = Used for conversion into oil, explosives and fuel.
+item.spore-pod.name = Bulbe sporifère
+item.spore-pod.description = Utilisé pour l'obtention d'huile, d'explosifs et de carburants
```
Comment (itskatt) [inline core/assets/bundles/bundle_fr.properties]:
```suggestion
item.spore-pod.description = Utilisé pour l'obtention d'huile, d'explosifs et de carburants.
```

**Comment by JustYanns** [comment]
It looks like your branch is outdated, you should update it :p
I'd like to contribute to this draft, but I let you update it first, it's yours

**Comment by Anuken** [comment]
Closing due to inactivity; ping me to reopen if needed.

**Pull request closed without merging**